tests/suites/test_suite_pkcs11_client.data

```diff
@@ -13,3 +13,20 @@ pk_generate_sign:MBEDTLS_PK_ECDSA
 PKCS#11 RSA generate and sign
 depends_on:MBEDTLS_PK_C:MBEDTLS_RSA_C
 pk_generate_sign:MBEDTLS_PK_RSA
+
+PKCS#11 ECDSA generate, sign and verify with Cryptoki
```

Comment:
Having a test that generates a key in the token is good, but key generation is slow, and we don't need a lot of tests for key generation. So I think this test should be an import-sign-verify test rather than generate-sign-verify.